channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/JaicpReplies.kt

```diff
@@ -97,6 +98,7 @@ class HangupReply(val state: String? = null) : Reply("hangup") {
 @Serializable
 class TelephonySwitchReply(
     val phoneNumber: String? = null,
+    val method: TelephonySwitchMethod = TelephonySwitchMethod.INVITE,
```

Comment:
The addition of parameter in between of arguments will break compatibility
I'd suggest either move new parameter to the end of arguments list, or add backward compatible constructor without `method` argument and mark it as deprecated

Reply:
@nikvoloshin Thanks, figured out, moved the new argument to the end